tests/storage/clusterlock_test.py

```diff
@@ -24,13 +24,14 @@
 import json
 import time
```

Comment:
While you touch this file, it will be useful to add the missing blank line before pytest.